ui/src/app/store/bootresource/slice.ts

```diff
@@ -13,19 +19,90 @@ const bootResourceSlice = createSlice({
   initialState: {
     connectionError: false,
     eventErrors: [],
+    fetchedImages: null,
     otherImages: [],
     rackImportRunning: false,
     regionImportRunning: false,
     resources: [],
     statuses: {
-      poll: false,
```

Comment:
This is totes fine, I just thought we discussed making these status names the same as the action, but maybe that was more musing that deciding.

Reply:
Oh, I thought we decided the opposite! I think this is what some would call a *complete and total breakdown of communication* hahaha. I actually like the idea of using the action name more, but maybe this is something we should update for all the models at once.

Added an issue here: https://github.com/canonical-web-and-design/maas-ui/issues/2767